docs/api/ReactWrapper/slice.md

```diff
@@ -0,0 +1,44 @@
+# `.slice([begin[, end]]) => ReactWrapper`
+
+Returns a new wrapper with a subset of the nodes of the original wrapper, according to the rules of `Array#slice`.
+
+
+#### Arguments
+
+1. `begin` (`Number` [optional]): Zero-based index from which to slice (defaults to `0`).
+2. `end` (`Number` [optional]): Zero-based index at which to end slicing (defaults to `length`).
```

Comment:
this isn't entirely how slice works - slice supports negative indexes as well. https://tc39.github.io/ecma262/#sec-array.prototype.slice has language that might be helpful.

Also, "index" implies "zero-based" - it's zero-based in virtually every language.